pkg/cmd/destination.go

```diff
@@ -0,0 +1,252 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog/v2"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	scribev1alpha1 "github.com/backube/scribe/api/v1alpha1"
+)
+
+var (
+	scribeNewDestinationLong = templates.LongDesc(`
```

Comment:
These come out as pretty big walls of text. Could they be line wrapped like https://github.com/kubernetes/kubectl/blob/0149779a03735a5d483115ca4220a7b6c861430c/pkg/cmd/get/get.go#L88

Reply:
will do

Reply:
done